packages/@rnw-scripts/format-files/src/formatFiles.ts

```diff
@@ -21,7 +21,7 @@ import {getNativeBinary} from 'clang-format';
 
 /// These constants control which files are formatted
 const includeEndsWith = ['.h', '.cpp'];
-const excludePathContains: string[] = [];
+const excludePathContains: string[] = ['vnext/codegen'];
```

Comment:
clang-format allows overriding rules for specific directories. Some directories have a clang-format config that disables rules for that directory.

Consider using this config-file based override instead of changing the tooling. The advantage is that raw clang-format, or tooling aware of autoformat, would also avoid formatting the directory, without needing to specialize the runner script